packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/appatidp.md

```diff
@@ -4,6 +4,8 @@
 
     For example, if your Okta subdomain is called `company`, then the URL would be: `https://company.okta.com/oauth2/v1/authorize/callback`. If you have configured a custom domain in your Okta Org, use that value to construct your redirect URI, such as `https://login.company.com/oauth2/v1/authorize/callback`.
 
+    > **Note:** If you set `issuerMode` to `DYNAMIC` [mode](/docs/reference/api/idps/#property-details), then you must add both the Okta URI and the custom URI in the external IdP's allowed `redirectUri` list.
```

Comment:
I think you need to add context here since this section shows up first, users don't know where `issuerMode` is configured.

```suggestion
    > **Note:** If you set the `issuerMode` parameter to `DYNAMIC` [mode](/docs/reference/api/idps/#property-details) when you [create an identity provider in Okta](#create-an-identity-provider-in-okta), then you must add both the Okta URI and the custom URI in the external IdP's allowed `redirectUri` list.
```
Also, should we add `issuerMode` parameter description in the "Create an identity provider in Okta" section? We currently don't describe this parameter from the Admin Console.